tc/core/polyhedral/reduction_matcher.cc

```diff
@@ -90,11 +90,8 @@ bool pwAffInvolvesOnlyInputDim(isl::pw_aff pa, int redDimIdx) {
   return true;
 }
 
-// Does pa have the form S(...) -> [(K*r)] where S is either a reduction init
+// Does pa have the form S(...) -> [f(r)] where S is either a reduction init
```

Comment:
actually, a "reduction init" does not exist anymore so this function is not detecting it